skills_ml/algorithms/occupation_classifiers/__init__.py

```diff
@@ -10,11 +10,13 @@
 from typing import Generator
 import logging
 
+onet = Onet()
+
+all_soc = onet.all_soc
```

Comment:
Ideally importing a module should be a lightweight action and not include stuff like this. Would it be so bad to put this in the constructor for FullSOC?

Reply:
you're right. I forgot to put it back to the constructor, but one thing I do wanna change is to ignore onet object when serializing like what we do to the model.

Reply:
actually we don't even need to have an attribute for onet object. we just need to call it once and get all the soc codes in `self.choices`.